docs/tasks/local-github-cache.md

```diff
@@ -0,0 +1,225 @@
+# Run a local GitHub Cache
+
+The cache for GitHub Actions can speed up CI/CD pipelines but using GitHub's hosted cache can (sometimes) be slow or even lock up entirely on self-hosted runners.
+
+GitHub runs VMs and the accompanying cache on the same network, so they can talk over a high speed backbone connection. When you run a self-hosted runner any download or upload operations to the cache are taking place over the public Internet.
+
+To improve cache speeds with Actuated runners you can run a self-hosted S3 server and switch out the official [actions/cache@v3](https://github.com/actions/cache) with [tespkg/actions-cache@v1](https://github.com/tespkg/actions-cache). The tespkg/actions-cache@v1 can target S3 instead of the proprietary GitHub cache.
+
+If you are only running a single Actuated agent it is possible to run the S3 cache directly on the agent host for ultimate near-loopback speed. If you have multiple Actuated runners the best option is to run the cache on a machine in the same region and network as the runners.
+
+## Set up an S3 cache
+
+There are a couple of options to run a self-hosted S3 server, most notably [Seaweedfs](https://github.com/seaweedfs/seaweedfs) and [Minio](https://min.io/).
+
+This guide will cover the setup of Seaweedfs but any S3 compatible service will work.
+
+### Install Seaweedfs
+
+Seaweedfs can be installed with [arkade](https://github.com/alexellis/arkade).
```

Comment:
Seaweedfs is distributed as a static Go binary, so it can be installed with arkade, or from the [GitHub releases page....]